reference/forms/types/birthday.rst

```diff
@@ -42,6 +42,7 @@ option defaults to 120 years ago to the current year.
 |                      | - `invalid_message`_                                                          |
 |                      | - `invalid_message_parameters`_                                               |
 |                      | - `mapped`_                                                                   |
+|                      | - `help`_                                                                     |
```

Comment:
should be between `invalid_message_parameters` and `mapped` IMO to keep options in alphabetical order

Reply:
same in the other files